src/app/components/control-bar/control-bar.component.ts

```diff
@@ -376,319 +377,117 @@ export class ControlBarComponent implements OnInit {
     this.geoDataService.setActiveGroupFeature(this.activeGroupFeature);
   }
 
-  // saves project as a CSV file by first organizing a JSON or a CSV and converting it. Saves to either MyData or local
-  // I apologize in advance for this mess of a function -Ben
-  // This really needs to be split into something like 3 separate functions
+  getCSVHeaders(exportFeatures) {
+    let headerArray = [];
+    exportFeatures.forEach((ef) => {
+      headerArray = [...Object.keys(ef)];
+    });
+    return headerArray;
+  }
+
+  exportList() {
```

Comment:
Exporting only exports a csv for each group. What about images not in a group

Could we add an all.csv  which has the following:
```
src,id,longitude,latitude,groups
```

Where groups column has a qouted value of group namas (i.e. https://stackoverflow.com/questions/4617935/is-there-a-way-to-include-commas-in-csv-columns-without-breaking-the-formatting).  So something like:
```
src,id,longitude,latitude,groups
https://agave.designsafe-ci.org/geo/v2/assets/836/e740a65b-0f6c-4ca3-896c-fd40403c002d.jpeg,587360,-80.780375,32.6185055555556,"foo-group,bar-group"
```

Reply:
Added a ticket https://jira.tacc.utexas.edu/browse/DES-2446